### ISSUE TITLE
The location can be got from Network.

### ISSUE THREAD

**Issue by HOLDfoot** [obaro/SimpleLocationApp#1]
Hi,
&nbsp;&nbsp;&nbsp;&nbsp;When I am finding a completely app that can run successfully. I find your profile of the demo. And I can run it with GPS successfully. But the location can not be accessed by network way. Finally, I found the INTERNET permission is not added to AndroidManifest.xml. 
&nbsp;&nbsp;&nbsp;&nbsp;Please help to confirm it. Finally, your demo helps me.</br>
&nbsp;&nbsp;&nbsp;&nbsp;Thanks.

**Comment by obaro** [comment]
Yes @HOLDfoot , you need to declare the INTERNET permission in the manifest.

Also remember that for newer devices, you will have to check if the user has granted permission in code.
